examples/211-dotnet-WebClient-Intent-Detection/Program.cs

```diff
@@ -0,0 +1,191 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+#pragma warning disable CS8602 // memory is initialized before usage
+#pragma warning disable CS0162 // unreachable code is managed via boolean settings
+
+using Microsoft.KernelMemory;
+
+/* Use MemoryWebClient to run the default import pipeline
+ * deployed as a web service at "http://127.0.0.1:9001/".
+ *
+ * Note: start the Kernel Memory service before running this.
+ * Note: if the web service uses distributed handlers, make sure
+ *       handlers are running to get the pipeline to complete,
+ *       otherwise the web service might just upload the files
+ *       without extracting memories. */
+public static class Program
+{
+    private static MemoryWebClient? s_memory;
+    private static readonly List<string> s_toDelete = new();
+
+    public static async Task Main()
+    {
+        s_memory = new MemoryWebClient("http://127.0.0.1:9001/");
+
+        // =======================
+        // === INGESTION =========
+        // =======================
+
+        // await StoreIntent();
```

Comment:
this call shouldn't be commented out. 

Inside `StoreIntent()`, you can use constant `docId`s and `IsDocumentReadyAsync` to check if the intents are already stored

Reply:
I'm planning to leave this as is, only uncomment line 29.
@dluc

Reply:
done